### ISSUE TITLE
open-proposal-weeecycle

### ISSUE THREAD

**Issue by gaiagreentech** [filecoin-project/devgrants#950]
# Open Grant Proposal: `WEEECycle: Tokenization of 10 tons of e-waste pegged to document storage on Filecoin`

**Name of Project:** `WEEECycle, powered by GAIA GreenTech`

**Proposal Category:**  `Green` 

**Proposer:** `gaiagreentech`

**Technical Sponsor:** `Alan Ransil`

**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT, APACHE2, or GPL licenses?:** `No` 

# Project Description

**WEEECycle, powered by GAIA GreenTech**

GAIA GreenTech is an intelligent **reverse logistics management platform for waste electrical and electronic equipment (WEEE)** that provides an end-to-end solution to the growing problem of incorrect disposal of electronic waste.  Through the I**PFS technology stack**, the implementation of a **proprietary traceability system,** rewards for incentives for correct disposal with **GAIA points,** and the **tokenization of waste collected (WEEECycle token),** we offer a WEEE management solution for manufacturers and consumers, in accordance with the regulatory requirements of each country. 

### The GAIA system is based on **3 pillars**:

1. **<span style="text-decoration:underline;">Reward system for consumers:</span>** In Latin America and countries like Brazil, the recycling rate of WEEE is less than 2%. In countries with the most advanced recycling systems,[ less than 20% of e-waste is formally recycled](https://www.unep.org/news-and-stories/press-release/un-report-time-seize-opportunity-tackle-challenge-e-waste). People don't know where to dispose of it and **have no incentive to do so.** We use an established network of business partners to enable users to convert their e-waste into consumption points that can be used in a variety of stores, both physical and digital. People can also choose to donate the points to social and environmental programs, such as S.O.S Amazon. The benefit of the points system is that, through the partner network, we can triple the value returned to customers. Through the reward system, we create an incentive mechanism for correct disposal, enabling a rapid increase in WEEE recycling rates.
2. **<span style="text-decoration:underline;">Traceability module and decentralized storage of information: </span>** the system will be designed so that all WEEE collected by GAIA is tracked by the GAIA system from disposal to sending to approved recyclers, optimizing the data collected. One of the biggest challenges in the industry is proving that electronics come from individual consumers, that's how we solve this problem. The data extracted by the system and the documents necessary **to legally prove the correct destination of WEEE are registered in the IPFS system**, ensuring the traceability of electronic waste batches and mitigating the risks related to tampering or loss of the system information.
3. **<span style="text-decoration:underline;">Tokenization of WEEE with WEEECycle Token: </span>** GAIA digitalized all documents related to the **WEEE Final Disposal process on the blockchain and then mints WEEECycle tokens** tied to the information representing the mass of e-waste correctly destined to recycling centers. WEEECycle can be used by e-waste manufacturers to meet their quantitative targets and environmental compliance deadlines of the Sector Agreement.

The use of IPS aggregated with the Gaia system, allows the creation of an immutable, decentralized, easy, and **elegant proof of the correct disposal of WEEE**, that can be verified by public authorities, verifiers, transporters, or final consumers. This process **reduces the bureaucracy of the system** and at the same time **avoids the collision and double spending of electronic invoices associated with e-waste, which is one of the biggest problems in the recycling industry.**

### **<span style="text-decoration:underline;">Real use case and really important information:</span>** 
much is said about the ability of blockchain, decentralization, and web3 to cause major disruptions. This is incredible. But as long as the industry is closed in on itself, the impact will not be achieved. We need to get out, face the streets, and develop real and legally valid cases. In Brazil and in developing countries, basic information is lacking, even about recycling rates. We can change this together. **The GAIA system takes Filecoin to the home of every person who has an old cell phone stored away, and to every regulator** who says that web3 is just speculation and good for nothing. Real cases with positive impact bring **huge benefits to the Filecoin ecosystem** and web3 as a whole. 

### **<span style="text-decoration:underline;">Reuse Frist and Digital Inclusion:</span>** 
GAIA system and its ecosystem of partners are designed to extract the most positive value from the reverse logistics system. The reuse of devices is always prioritized and part of the electronics that can be re-introduced to the market is destined for NGOs and Digital Inclusion Center.

### **<span style="text-decoration:underline;">Operational Company in partnership with an exchange with over 3 million users</span>**:
 GAIA is an operational startup, with a viable business model, that **has already collected dozens of tons of electronic waste,** completed its first investment round, and is in the process of expanding its operation. The sale of WEEECycle tokens will be carried out with a partner that already has a database with millions of users. You won't find this every day on the web3. 

### **<span style="text-decoration:underline;">Defined Buyers target & demand far exceeding supply scenario: </span>** 
unlike almost all web3 tokens and projects, WEEECycle tokens have an **established end buyer and demand far exceeding available supply.** The recycling target of electronic manufacturers is **tens of times larger than the current recycling capacity** of countries, creating a really strong buying force. At the same time, [e-waste is the fastest growing waste stream in the world.](https://ewastemonitor.info/gem-2020/) The more tons we can tokenize, the greater the interest manufacturers have in our offer. **They want thousands of tons and we will deliver.**

### **<span style="text-decoration:underline;">An incentive to enter the web3 with a simple and fun contact:</span>** 
any companies and individuals interested in using the WEEECycle token for their compensation will need to burn their tokens. At that point, **users have received an NFT that links the token and the offset to a specific batch of WEEE**, with meta-data linking the documents stored in Filecoin. **NFTs have already proven to be an interesting onboarding tool for non-crypto users.** We will use them as a fun certificate that can be used as a compensation 'trophy' by companies and end users while allowing people to access IPFS in a simple and modern end-user way.

### **<span style="text-decoration:underline;">Turning fear and distrust into a nice example: </span>**
Social projects, end users, and large companies are afraid to venture into the web3. The GAIA system depends on an ecosystem that involves regulators, recyclers, partner stores, end users, and large electronics manufacturers. **The success of the GAIA system proves how web3 and decentralized data storage can be used by anyone and has value for different stakeholders,** not just the degens of the metaverse. We want to inspire everyone to use and interact with Filecoin

## Deliverables

The deliverables relate to the steps necessary to **tokenize 10 tons of e-waste collected by GAIA**, sell the tokens representing those tons, and claim the NFTs for compensation, with end-to-end registration in the Filecoin system.

* **Integration of GAIA's backend and App with the Filecoin system:** the creation of the framework for making available the database collected at the time of disposal of the electronic waste in Filecoin, as:
    * WEEE line
    * Device type
    * Device image
    * Collection location
    * Collected quantity
    * Logistics operator
    * Destination location
    * Device status
    * Amount of recycled material
* **Storage in the IPFS of the documents necessary for legal proof of the correct destination of the electronic waste**, which will be used to create the WEEECycle tokens, including: 
    * **Waste Transportation Manifest (MTR):** the transportation of WEEE is subject to legally certified processes. For each shipment and transport of WEEE volume, it is necessary to fill out the Waste Transportation Manifest (MTR) in the National Information System of Solid Waste Management (SINIR) and State System of Online Solid Waste Management (SIGOR). The MTR is the system that identifies the generator, transportation provider, disposer, and the type of waste being transported. It creates a code that proves the legality of the transported volume and ensures the stakeholders are involved in the process. 
    * **Electronic Invoice (NF-e)**: In addition to the MTR, the generator of the WEEE issues an NF-e to the recycler for the waste. NF-e works as a protection for both parties in the system.
    * **Certificate of WEEE Receipt and Final Disposal (CDF)**: The recipient company (recycler) sorts the electronic waste delivered by the transporter, validating the information registered in the MTR by the waste generator. The approval of the MTR information in SINIR and SIGOR by the receiver will automatically generate the Certificate of Final Disposal (CDF), which will be used for the governmental system.
* **Creation of an interface/marketplace where users can burn their tokens** and receive/claim an NFT linked directing the user to Filecoin storage so that they can check for themselves the lots associated with their compensation.
* **Content Creation:** E-waste is a reality for billions of people and only a small percentage of them know how to interact with web3, tokens, and NFTs. We will need to create learning paths and tutorials that teach in an easy and non-technical way what they need to know to take advantage of the GAIA and Filecoin solutions.
* **Workshop and working sessions with leaders from the government, recyclers, and electronics manufacturers:** the success of the project essentially depends on making the technology friendly to certain stakeholders. Even if the system technically fulfills all the legal criteria, if we don't have the sympathy of decision makers, the path will be painful. Therefore, we will reserve a relevant part of our time to talk and explain to each decision maker how/why the decentralized storage system works and get their trust.

## Development Roadmap

1. **Pre-production and Detailed Roadmap**

Phase is intended for internal alignment with the development team to detail the workflow necessary for the integration of IPFS and Filecoin with the entire GAIA system and for creating the necessary interfaces to talk to the end customer. We also leverage the defined architecture to create the materials and presentations needed to start conversations with decision-makers.

Staff: 4 programmers. + 1 product manager + 1 ecosystem developer

Cost: $4,000

Duration: 1 weeks

2. **Database integration**

Proof of concept for the creation of an informational standard that allows the integration and decentralized storage in the IPFS of the documents collected and processed by the Gaia traceability system in the collection of electronic waste, until the registration of the Certificate of WEEE Receipt and Final Disposal (CDF) provided by recyclers. The deliverable here is to prove that integration is feasible within the operation of a reverse logistics manager. For this purpose, we plan to use ec2 instances of AWS for developing an API IPFS to submit the all documentation

Staff: 4 programmers + 1 product manager 

Cost: $17,000.

Duration: 4 weeks

3. **Tokenization of information to create the WEEECycle:**

With all the integration completed and the documentation properly stored on-chain, the supply of WEEECycle will be minted and backed by the registered documentation. By function, new WEEECycles can only be created when new batches of information have been duly registered in Filecoin, proving the correct disposal of electronic waste. In the development of the WEEECycle Token, we plan to use AWS gateway, Cloudflare, Cognito, EKS, AWS gateway, Cloudflare, Cognito, EKS, MongoDB, Polygon, and KMS.

Staff: 4 programmers 

Cost: $ 12.000.

Duration: 3 weeks

4. **Interface/Marketplace for Compensation and Claim NFTs**

Development of NFTs marketplace where users burned their WEEECycle to receive/claim documents that represent the correct destination of electronic waste that allows the compensation of electronic waste by the end users or manufacturer. The marketplace will have the following functions: social login and wallet integration, WEEECycle burn, My NFTs Space, and My benefits Space. After burning WEEECycle, users will be able to claim an NFT (representing the burned portion of WEECycle) that contains the document metadata stored in the IPS that proves the reverse logistics process.

Staff: 4 programmers + 1 product manager + 1 website design 

Cost: $22,000.

Duration 5 weeks

5. **Content Creation**

Creation of tutorials, presentations, and videos necessary for meetings with decision-makers, users who are going to buy WEEECycle, and users who want to claim their e-waste compensation. 

One Workflow presentation explained all the processes

One detailed video tutorial explained all the solutions

Five small video presentations teaching users to:

* What is GAYA Ecosystem and how to use the GAIA app
* Why tokens are legally valid
* Buy WEEECycle Token
* What and how to claim WEEEcycle compensation
* How to redeem NFT and check the information inside Filecoin

All videos will be published on GAIA's social media and will be widely publicized.

Staff: 1 content creator

Cost: $5,000

Duration 160 hours

* **Workshops with leaders from the government, recyclers, and manufacturers.**

Detailed presentation of GAIA's operation flow and Filecoin's distributed storage system, in order to captivate decision makers and explain to each of them how the system works.

Staff: Gaia CEO + programmer

Cost: $2,000

Duration: 6 weeks

## **Visual timeline**

![alt_text](https://gaia-static-files.s3.us-east-2.amazonaws.com/Visual+timeline.png "image_tooltip")

## Total Budget Requested

<table>
  <tr>
   <td>
   </td>
   <td>Deliverable
   </td>
   <td>Price
   </td>
   <td>Duration
   </td>
  </tr>
  <tr>
   <td>1.
   </td>
   <td>Pre-production 
   </td>
   <td>$4,000
   </td>
   <td>1 week
   </td>
  </tr>
  <tr>
   <td>2.
   </td>
   <td>Database integration 
   </td>
   <td>$17.000
   </td>
   <td>4 weeks
   </td>
  </tr>
  <tr>
   <td>3,
   </td>
   <td>Tokenization of information
   </td>
   <td>$12,000
   </td>
   <td>3 weeks
   </td>
  </tr>
  <tr>
   <td>4.
   </td>
   <td>User interfaces / Marketplace
   </td>
   <td>$22,000
   </td>
   <td>5 weeks
   </td>
  </tr>
  <tr>
   <td>5.
   </td>
   <td>Content Creation
   </td>
   <td>$5,000
   </td>
   <td>160 hours
   </td>
  </tr>
  <tr>
   <td>6.
   </td>
   <td>Workshop
   </td>
   <td>$2,000
   </td>
   <td>6 weeks
   </td>
  </tr>
  <tr>
   <td>
   </td>
   <td>Total
   </td>
   <td>$62,000
   </td>
   <td>
   </td>
  </tr>
</table>

US$ 62.000 per 12 weeks of work

## Maintenance and Upgrade Plans

# Team

### **Team, Experiences, and Github**

[Thalita Braga](https://www.linkedin.com/in/thalita-braga/) - Co-founder & CEO at GAIA GreenTech

Thalita co-founded GAIA GreenTech in 2018 while graduating in Material and Manufacturing Engineering from the University of São Paulo (USP). She has an additional diploma in Training and Roadmapping Technology for Researchers and Entrepreneurs working with Glass and has a Bachelor in Agricultural Engineering. 

[Raúl Julián Revelo PhD](https://www.linkedin.com/in/raulrevelo-9969032a/details/experience/) - Co-founder at GAIA GreenTech

He has a Master of Science and Ph.D. in Material Science Engineering from the University of São Paulo, and a Bachelor's Degree in Material Engineering from the University of Antioquia. With 6 years of experience in Academic Research, and 2 years working with projects that generate Carbon Certificates.

[Diogo Puppim de Oliveira ](https://www.linkedin.com/in/diogo-puppim-de-oliveira-207905a7)- Head of Technology | SRE | DevOps

SRE, DevOps and Master Degree in Computer Science Research Area: Artificial intelligence. Skilled in Architecture planning, Monitoring of production systems, Management of AWS cloud environment, Process of CI/CD, Infra as a code, Python/Django, React, React Native, JavaScript, TypeScript, machine learning with Pandas, Keras, Numpy, Scipy, Scikit-learn andMatplotlib.

[Henrique Leite ](https://www.linkedin.com/in/henrique-leite-2903)– Blockchain Developer | DeFi | Smart Contracts 

Blockchain developer and architect with 16 years of experience in technology and Industry. Participated in large consultancies and worked on projects for the main banks in Brazil, AgroTech, and Fintech. He is the founder of GoBlockChain, where works in smart contracts and, DeFi.

[Pietro Martins](https://www.linkedin.com/in/profpietromartins) - WEB3/Blockchain Tech Lead | Developer

Blockchain developer, consultor and instructor with a Master’s Degree focused on Computer Science. Experienced Information Technology Developer and High Education Teacher with a demonstrated history of working in IT projects. Skilled in C, C++, C#, Java, Python, HTML, CSS, JavaScript, Visual Studio, OpenCV, Solidity, Ethereum, EOS.IO, and blockchain in general.  

[Rafaela Romano](https://www.linkedin.com/in/rafaelaapromano/) - Strategist | Writer | PR  | Marketing

Master in Anthropology from the University of Sao Paulo, and she was previously editor-in-chief at Cointelegraph Brazil, the largest crypto portal in the world. Rafaela was elected one of the 50 most influential people in the Brazilian crypto market and is the Co-founder of Impacta Finance, in which she concretizes her studies and practice on the relationship between blockchain and sustainability after 3 years of intensive research.

[Paula Palermo](https://www.linkedin.com/in/paula-palermo/) - Ecosystem Growth Strategist   

Architect, urban planner, and economist. Since 2016 she has worked with collaborative local development and since 2018 with blockchain for social impact projects, accumulating over 15 years of additional volunteer experience in social initiatives. She was previously part of the Ventures teams at Plug and Play Ventures and Techstars, working with startups and corporations on the topics of Web3, Smart Cities, and Sustainability.

## Team Website

[https://www.gaiagreentech.com/](https://www.gaiagreentech.com/)

## Team code repositories

[https://github.com/falehenrique](https://github.com/falehenrique)

[https://github.com/goblockchain](https://github.com/Puppim)

[https://github.com/Puppim](https://github.com/Puppim)

**Comment by ErinOCon** [comment]
HI @gaiagreentech, thank you for your proposal. This grant has been approved! Can you confirm the preferred email address for discussing next steps?

**Comment by gaiagreentech** [comment]
Hello,
We are very excited and happy with the news. Looking forward to starting a
great positive impact work together with Filecoin, for a greener and fairer
future.😄🌎💚

For the next steps, let's focus the informations on Paula and Rafaela.
***@***.*** and ***@***.***
And I can be copied in emails to collaborate on work:
***@***.***

Best regards,
*Thalita Braga*
GAIA GreenTech
CEO & Co-founder

+55 (16) 99628-5289
***@***.***
gaiagreentech.com




Em sáb., 17 de set. de 2022 às 18:56, ErinOCon ***@***.***>
escreveu:

**Comment by ErinOCon** [comment]
Thank you, @gaiagreentech!
